### DISCUSSION TITLE
Forced on Maximum Resolution on video playback.

### DISCUSSION THREAD

**Discussion by avneroi** [PoomSmart/YTABConfig#84] [Q&A]
Is there any flag on A/B to force the resolution to be always 1080p (1440p or 2160p in rare cases)? Thanks!

**Comment by microfx** [comment]
looking for an answer as well... nobody?

**Comment by PoomSmart** [comment]
YTChooseVideoQuality tweak.

**Reply by microfx** [reply]
Where can I find it? It's not in A/B or uYou settings hmm do I need to download something?

**Reply by PoomSmart** [reply]
I said a tweak.
